### DISCUSSION TITLE
HTTPS in production ?

### DISCUSSION THREAD

**Discussion by kevhof** [nuxt/nuxt#18091] [Questions] [answered]
I know there is a way to use https in development ([here](https://v3.nuxtjs.org/getting-started/commands#development-server)). But what about production / nuxi build ?

**Comment by pi0** [comment, marked as answer]
Hi! You probably need a reverse proxy like Nginx or Cloudflare to support SSL in production. Please track https://github.com/nuxt/framework/issues/1701 for more info.

**Reply by kevhof** [reply]
I didn't know about nginx as a ssl reverse proxy. Thanks.

For those interested there is a [guide](https://www.sitepoint.com/configuring-nginx-ssl-node-js/).
